### PR TITLE
Added Mobile App support

### PR DESCRIPTION
**Pull request by NgLoader**
Supporting the new pterodactyl mobile app

https://pterodactyl.cloud

How to enable: set `MOBILE_APP` to true in `.env`

### CODE REVIEW

File: manifest/panel/entrypoint.sh [hunk 1]
```diff
@@ -31,7 +31,8 @@ function init {
     ln -s /data/pterodactyl.conf .env
 }
 
-# Runs the initial configuration on every startup
+# Runs the 
+ial configuration on every startup
```
Comment (Derkades) [inline manifest/panel/entrypoint.sh]: Accidental?

File: manifest/panel/entrypoint.sh [hunk 2]
```diff
@@ -66,10 +66,11 @@ function startServer {
     fi
 
 
-    # is mobile app enalbed
+    # CHecking is mobile app enalbed
```
Comment (Derkades) [inline manifest/panel/entrypoint.sh]: maybe replace with `Check if the mobile app is enabled`

File: manifest/panel/entrypoint.sh (hunk 1 above)
Comment (NgLoader) [reply manifest/panel/entrypoint.sh]: yea

File: manifest/panel/entrypoint.sh (hunk 2 above)
Comment (NgLoader) [reply manifest/panel/entrypoint.sh]: thanks

**Pull request closed without merging**